drawbicyc/include/drawbicyc/PlotterMicro.hpp

```diff
@@ -308,10 +308,10 @@ class PlotterMicro_t : public Plotter_t<NDims>
                      * 1e3;        // to mm
   }
   // accumulated volume precipitation [m^3]
-  double calc_acc_volume_precip(double prec_vol)
+  double calc_acc_surf_precip__volume(double prec_vol)
   {
     if(this->micro == "lgrngn")
-      return prec_vol / this->DomainVolume;
+      return prec_vol / this->DomainSurf * calc_acc_surf_precip(prec_vol);
```

Comment:
it should be:
return calc_acc_surf_precip(prec_vol) * this->DomainSurf / 1000.

regardless of the microphysics used (lgrngn or blk_1m)